Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/MediaControl.swift

```diff
@@ -218,13 +218,24 @@ open class MediaControl: UICorePlugin, UIGestureRecognizerDelegate {
         self.view.bindFrameToSuperviewBounds()
     }
 
-    func renderPlugins(_ plugins: [UICorePlugin]) {
-        plugins
-            .filter { $0 is MediaControlPlugin }
-            .map { $0 as! MediaControlPlugin }
-            .forEach { plugin in
+    func renderPlugins(_ plugins: [MediaControlPlugin]) {
+        if let mediaControlPluginOrderOption = core?.options[kMediaControlPluginsOrder] as? String {
+            let pluginsOrder = mediaControlPluginOrderOption
+                .replacingOccurrences(of: " ", with: "")
+                .components(separatedBy: ",")
+
+            var orderedPlugins: Array<MediaControlPlugin> = plugins.filter { pluginsOrder.contains($0.pluginName)}
+            orderedPlugins.append(contentsOf: plugins.filter { !pluginsOrder.contains($0.pluginName)})
+
+            orderedPlugins.forEach { plugin in
+                mediaControlView.addSubview(plugin.view, panel: plugin.panel, position: plugin.position)
```

Comment:
Maybe for later since other places use the exact same function call, but how about having another `addSubview` or even `addPlugin` function that takes a plugin instead of taking its properties? so we end up calling it like `mediaControlView.addPlugin(plugin)`

Reply:
We changed the parameters names only to have a more meaningful swifty names but add the method addPlugin to the view will give to view a responsibility out of view context so we preferred do not change it at this moment. Do you agree?

Reply:
Totally! Good point :+1: